### DISCUSSION TITLE
Problem with github action

### DISCUSSION THREAD

**Discussion by NetZach777** [pytest-dev/pytest#11624] [General]
Hi,

i push in github roberta model with 472mo 

but i have this problem

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
ERROR sentiment-analysis/tests/test_main.py - OSError: Can't load the configuration of '../model/roberta_model_V3'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure '../model/roberta_model_V3' is the correct path to a directory containing a config.json file
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
========================= 1 warning, 1 error in 8.20s ==========================
Error: Process completed with exit code 2.

from fastapi import FastAPI, HTTPException
from pydantic import BaseModel
from transformers import RobertaTokenizer, TFRobertaForSequenceClassification
import tensorflow as tf

app = FastAPI()

MODEL_PATH = r"../model/roberta_model_V3"
tokenizer = RobertaTokenizer.from_pretrained("roberta-base")
model = TFRobertaForSequenceClassification.from_pretrained(MODEL_PATH)

Thanks for your help

**Comment by haixuanTao** [comment]
I'm having the same problem. I think wonder if huggingface is available from github action.

See: https://github.com/dora-rs/dora/pull/640

**Comment by RonnyPfannschmidt** [comment]
closing as this is completely unrelated to pytest - pl ease ask at the correct projects
